source/_components/switch.synology_homemode.markdown

```diff
@@ -0,0 +1,34 @@
+---
+layout: page
+title: "Synology Homemode Switch"
+description: "Instructions on how to integrate Synology Homemode Switch"
+date: 2018-01-31 16:00
+sidebar: true
+comments: false
+sharing: true
+footer: true
+logo: Synology.png
+ha_category: Switch
+ha_release: 0.66
+ha_iot_class: "Local Polling"
+---
+
+The `synology_homemode` platform allows you to switch the home mode of your [Synology](https://www.synology.com/) Surveillance Station in Home Assistant.
+
+To enable your Surveillance Station Home Mode in your installation, add the following to your `configuration.yaml` file:
+
+```yaml
+# Example configuration.yaml entry
+switch:
+  - platform: synology_homemode
+    url: http://yousyno:5000
+    username: xxxxxxxx
+    password: xxxxxxxx
+```
+
+Configuration variables:
```

Comment:
Please use the configuration tags, for more information please see:
https://home-assistant.io/developers/documentation/create_page/#configuration